src/views/List.vue

```diff
@@ -52,6 +53,20 @@ export default class List extends Vue {
   protected dropzoneRef = 'dropzone'
   protected dropzone!: Dropzone
 
+  private mounted() {
+    const flashesModule = getModule(FlashesModule, this.$store)
+    const metaMaskModule = getModule(MetaMaskModule, this.$store)
+    const web3Module = getModule(Web3Module, this.$store)
+
+    if (typeof ethereum === 'undefined' || typeof ethereum.selectedAddress === 'undefined') {
```

Comment:
Having the same exact code twice is serious code smell and indicates the logic isn't correct.  Remove this and rely on a single correct block as per the comment above.